src/runtime/contrib/cublas/cublas.cc

```diff
@@ -36,6 +36,36 @@ inline cublasOperation_t BooleanToTranspose(bool item) {
   return item ? CUBLAS_OP_T : CUBLAS_OP_N;
 }
 
+inline void TryEnableTensorCore(cublasHandle_t hdl) {
+  // TensorCores are only supported in cublas 9.0 or higher
```

Comment:
Also cuda compute capability is 70 or higher? `__CUDACC_VER_MAJOR__ >= 7`

Reply:
Cublas and CuDNN will check it. If we try to use TensorCores on non-TensorCore GPUs, it will fallback to the traditional workload. So I think there is no need to duplicate this check in tvm.